frequency-chain/style-guide
# add svelte-check and resolve all errors

# Problem
When running svelte-check, we're getting lots of errors

# Solution
Configure our project to resolve each of the errors until when running `npm run check` we get 0 errors.

## Steps to Verify:

1. Checkout branch
2. Run `npm run check`
3. See 0 errors
